### PR TITLE
Use better BIDI examples.

### LINKED ISSUE #75: More about directions
In [§2.5.1 Quoted Literals](https://www.w3.org/TR/rdf12-turtle/#turtle-literals) as part of the examples, it may be better to use a "real" example for directions (the ones listed among the examples are really superfluous). E.g.,

```turtle
show:218 show:blurb "HTML و CSS: تصميم و إنشاء مواقع الويب"@ar--rtl.
```

or:

```turtle
show:218 show:blurb "HTML היא שפת סימון."@he--rtl
```

Maybe it is also important add a note whereby the text the value field in turtle represents the text as it is stored in memory and not as it is supposed to be displayed. (We spent quite some time to get it right in the spec for the [publication manifest](https://www.w3.org/TR/pub-manifest/#value-localizable-string), feel free to copy and paste.)

(In my experience, getting the spec examples right bidirectional text is a challenge; you should probably ask the i18n guys to check it.)

### PR DESCRIPTION
**Pull request by gkellogg**
Fixes #75.

***
<a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/pull/78.html" title="Last updated on Oct 28, 2024, 5:28 PM UTC (40b1247)">Preview</a> | <a href="https://pr-preview.s3.amazonaws.com/w3c/rdf-turtle/78/1e24486...40b1247.html" title="Last updated on Oct 28, 2024, 5:28 PM UTC (40b1247)">Diff</a>

### CODE REVIEW

**Review by iherman — APPROVED** [review]
We should flag this explicitly for the upcoming a11y review to see if this is all right. I am always messed up: should we see here what is displayed properly or what is stored in memory? I guess it is the former.

**Review by domel — APPROVED** [review]
Quite simple editions. Agree.

**Review by afs — APPROVED** [review]

File: spec/index.html [hunk 1]
```diff
@@ -521,13 +521,14 @@ <h3>Quoted Literals</h3>
...
-        show:218 show:localName "That Seventies Show"@en--ltr .            # literal with a language tag and base direction
+        show:218 show:localName "HTML היא שפת סימון."@he--rtl .            # literal with a language tag and base direction
         show:218 show:localName "That Seventies Show"@en-us--ltr .         # literal with a language tag, region subtag, and base direction
         show:218 show:localName 'Cette Série des Années Soixante-dix'@fr . # literal delimited by single quote
         show:218 show:localName "Cette Série des Années Septante"@fr-be .  # literal with a region subtag
         show:218 show:blurb '''This is a multi-line                        # literal with embedded new lines and quotes
-        literal with many quotes (""""")
-        and up to two sequential apostrophes ('').''' .
+                               literal with many quotes (""""")
+                               and up to two sequential apostrophes ('').''' .
```
Comment (TallTed) [inline spec/index.html:530]:
```suggestion
                               literal with many quotes (""""")
                               and up to two sequential apostrophes ('').''' . # literal with embedded new lines and quotes
```

File: spec/index.html [hunk 2]
```diff
@@ -521,13 +521,14 @@ <h3>Quoted Literals</h3>
         show:218 rdfs:label "That Seventies Show"^^<http://www.w3.org/2001/XMLSchema#string> . # same as above
         show:218 rdfs:label "That Seventies Show" .                                            # same again
         show:218 show:localName "That Seventies Show"@en .                 # literal with a language tag
-        show:218 show:localName "That Seventies Show"@en--ltr .            # literal with a language tag and base direction
+        show:218 show:localName "HTML היא שפת סימון."@he--rtl .            # literal with a language tag and base direction
         show:218 show:localName "That Seventies Show"@en-us--ltr .         # literal with a language tag, region subtag, and base direction
         show:218 show:localName 'Cette Série des Années Soixante-dix'@fr . # literal delimited by single quote
         show:218 show:localName "Cette Série des Années Septante"@fr-be .  # literal with a region subtag
         show:218 show:blurb '''This is a multi-line                        # literal with embedded new lines and quotes
```
Comment (TallTed) [inline spec/index.html:528]: Moving the comment *outside* the literal.
```suggestion
        show:218 show:blurb '''This is a multi-line
```

**Review by TallTed — CHANGES_REQUESTED** [review]

File: spec/index.html [hunk 3]
```diff
@@ -525,9 +525,9 @@ <h3>Quoted Literals</h3>
         show:218 show:localName "That Seventies Show"@en-us--ltr .         # literal with a language tag, region subtag, and base direction
         show:218 show:localName 'Cette Série des Années Soixante-dix'@fr . # literal delimited by single quote
         show:218 show:localName "Cette Série des Années Septante"@fr-be .  # literal with a region subtag
```
Comment (TallTed) [inline spec/index.html:527]: realign the right-side comments. Similar should be done to the 5 lines above these.
```suggestion
        show:218 show:localName "That Seventies Show"@en-us--ltr .           # literal with a language tag, region subtag, and base direction
        show:218 show:localName 'Cette Série des Années Soixante-dix'@fr .   # literal delimited by single quote
        show:218 show:localName "Cette Série des Années Septante"@fr-be .    # literal with a region subtag
```

**Review by TallTed — CHANGES_REQUESTED** [review]

**Pull request merged**